### PR TITLE
Added: Badge and Image

### PR DESCRIPTION
**Pull request by aiman-ansari**
Screenshot (14)
Screenshot (15)
Screenshot (16)
Screenshot (17)
Screenshot (18)
Screenshot (19)

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```
Comment (PuneetP16) [inline .vscode/settings.json:2]: add it in git ignore.

File: src/components/Badge/Badge.css [hunk 2]
```diff
@@ -0,0 +1,112 @@
+.badge-wrapper{
+ position:relative;
+ display:inline-block;
+}
+.badge{
+    border-radius:50%;
+    position: absolute;
+    background-color:var(--badge-online);
+    border:2px solid white
```
Comment (PuneetP16) [inline src/components/Badge/Badge.css]: semicolon missing. Use some formatting extensions like prettier as space between border and colon is not present.

File: src/components/Badge/Badge.css [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+.icon-badge{
+    position:absolute;
+    border-radius:50%;
+    height:1.2rem;
+    /* padding:0.5rem; */
+    width:1.2rem;
+    font-size:0.7rem;
+    top: -10px;;
+    right:-10px;
+    background-color:rgb(252, 59, 59);
```
Comment (PuneetP16) [inline src/components/Badge/Badge.css]: Add variables instead of direct color values.

File: src/components/Badge/Badge.css [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+    color:white
+}
+i{
+    /* font-size:5rem; */
```
Comment (PuneetP16) [inline src/components/Badge/Badge.css]: Commented code in many places, remove if not useful.

**Review by PuneetP16 — APPROVED** [review]
Looks good to merge.

File: src/components/Badge/Badge.css [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+}
+.secondary-btn{
+    background-color: var(--secondary-color);
+}
+.yellow-btn{
+    background-color:var(--theme-yellow)
+}
+.tertiary-btn{
+    background-color:var(--tertiary-color)
+}
```
Comment (ShrutiLakhani) [inline src/components/Badge/Badge.css:104]: Code looks clean.LGTM!

**Review by ShrutiLakhani — APPROVED** [review]

**Pull request merged**